.github/workflows/dependency-graph.yml

```diff
@@ -2,7 +2,7 @@
 name: Submit Dependency Graph
 on:
   push:
-    branches: [1.10.x] # default branch of the project
+    branches: [1.10.x, develop] # default branch of the project
```

Comment:
why is this needed?

Reply:
This is not strictly needed but Eugene recommended it when reviewing a similar PR

https://github.com/sbt/sbt/pull/7746#pullrequestreview-2357811045